go/examples/sign-request/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"os"
+	"strings"
+
+	tlsigning "github.com/Truelayer/truelayer-signing/go"
+	"github.com/google/uuid"
+)
+
+// the base url to use
+const (
+	TlBaseUrl = "https://api.truelayer-sandbox.com"
```

Comment:
Go usually uses full-upper acronyms if I remember correctly
```suggestion
	TLBaseURL = "https://api.truelayer-sandbox.com"
```